oca_dependencies.txt

```diff
@@ -32,5 +32,6 @@ stock-logistics-warehouse https://github.com/guewen/stock-logistics-warehouse wm
 # stock-logistics-warehouse https://github.com/guewen/stock-logistics-warehouse 12.0-add-stock_picking_zone
 # stock-logistics-warehouse https://github.com/guewen/stock-logistics-warehouse 12.0-add-reservation-zone-rules
 
-
-
```

Comment:
Please keep these empty lines to be able to agregate all the PRs in this repo :pray: